workers/META.yml

```diff
@@ -0,0 +1,13 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1071345
+      results:
+        - test: dedicated-worker-parse-error-failure.html
+    - product: chrome
+      url: https://crbug.com/906850
```

Comment:
This bug doesn't link to opaque-origin.html but rather a sandboxed-iframe-with-opaque-origin.html

Reply:
Interesting, it does get returned from monorail search (https://bugs.chromium.org/p/chromium/issues/list?q=%2Fworkers%2Fopaque-origin.html&can=2)... I wonder if monorail splits on `/` and I should be quoting these search terms -_-.

Reply:
Reverted